admin/all-features-except

```diff
@@ -55,4 +55,7 @@ if __name__ == "__main__":
     # drop default, as it may re-enable other features
     features.discard("default")
 
-    print("--no-default-features", "--features", ",".join(sorted(features)))
+    # NOTE: REMOVED `--no-default-features` as QUICK WORKAROUND for
+    # cargo-semver-checks which does not recognize this flag.
+    # (This happens to be OK in this fork which has no default features.)
+    print("--features", ",".join(sorted(features)))
```

Comment:
I think this update can be reverted, as I commented elsewhere - noting this before I forget